CHANGES/437.bugfix

```diff
@@ -0,0 +1 @@
+Gunicorn API workers default to 2
```

Comment:
This change makes sense to me, however I'd like first to understand how these 2 processes will access these https://github.com/pulp/pulp-operator/blob/main/roles/pulp-api/defaults/main.yml#L5

Reply:
@chambridge @tumido Can any of you answer that question by chance? i.e given the allocated resources, now that we have 1 process it has access to all of them, if the processes will be bumped to 2 or more, how do they share the allocated resources? They are evenly split or whoever gets first, takes the amount it needs and other risk to 'starve'?